.istanbul.yml

```diff
@@ -7,6 +7,7 @@ instrumentation:
     "public/**",
     "test/**",
     "coverage/**",
+    "src/defaultPhrases.js"
```

Comment:
Fair. I was thinking that we would ignore because writing tests for the file itself seems v. silly